packages/system-api/README.md

```diff
@@ -0,0 +1,27 @@
+# Genestack system APIs to use in browser
+
+## Installation
+```
+npm install @genestack/system-api
+```
+
+## Content
+### invokeMethod
+ Invoke a Java method from an application's class. Returns a Promise that is resolved with 
+ serialized/deserialized value that method returns or rejected wiht the Error object in case
+ of any failure.
+ 
+```typescript
+function invokeMethod(options: MethodInvocationOptions): Promise<any>
+```
+#### Method invocation options
+
+| Parameter           | Type             | Default | Comment                                          |
+|---------------------|------------------|---------|--------------------------------------------------|
+| `applicationId `    | string           |         | Id of application in format `"vendor/application"` |
+| `method`            | string           |         | Application method name                          |
+| `parameters`        | Array<JSONValue> |         | A list of parameters that method accepts. Each parameter should be JSON serializable|
```

Comment:
Unfortunately markdown tables seemed to be not supporting multiline tables.. =( I mean, in markdown. Render is ok but sources are ugly.